vignettes/production-percent-change.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Indicator Choices: Production vs. Percent Change in Production"
```

Comment:
Is "production" enough or do we need to qualify "production" with some word to more precisely distinguish it from "percent change in production"? Without a qualifier, *percent in* production seems like a subgroup of production when it is instead an alternative.

Three qualifiers that come to my mind are weighted, absolute, and raw.

Reply:
I elected to add "Weighted" in every case